src/index.html

```diff
@@ -7,8 +7,90 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        class="logo"
+        alt="home">
+      </a>
+
+      <nav class="navigation">
+        <ul class="fullList">
+          <li class="list">
+            <a href="apple"
+            class="nav_link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="list">
+            <a href="samsung"
+            class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="list">
+            <a href="smartphones"
+            class="nav_link">
+              smartphones
+            </a>
+          </li>
+
+          <li
+          class="list"
+          >
+            <a href="lapotp&computers"
```

Comment:
Change the href value to a valid URL or use # as a placeholder. Also, correct the typo 'lapotp' to 'laptops'.